### ISSUE TITLE
The KV endpoint: incorrect handler of response status code 403

### ISSUE THREAD

**Issue by fedotoff** [cablehead/python-consul#75]
KV.get handles response in callback.
If response has 403 status code (rpc error: rpc error: ACL not found) then we have ValueError: No JSON object could be decoded.

```python
 def callback(response):
                if response.code == 500:
                    raise ConsulException(response.body)
                elif response.code == 404:
                    data = None
               """Bug is here"""
                else:
                    data = json.loads(response.body)
                    if not keys:
                        for item in data:
                            if item.get('Value') is not None:
                                item['Value'] = base64.b64decode(item['Value'])
                        if not recurse:
                            data = data[0]
                return response.headers['X-Consul-Index'], data
```

**Comment by cablehead** [comment]
This should be fixed by #115

**Comment by cablehead** [comment]
Fixed by #115
